hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/token/NumericValidationTest.java

```diff
@@ -370,4 +389,314 @@ public Stream<DynamicTest> convertTinybarsToTinycents() {
                             .andAssert(txn -> txn.hasKnownStatus(testCase.status()))));
         }
     }
+
+    @Nested
+    @DisplayName("calls fail to non-static create/update token functions with invalid values")
+    class CreateAndUpdateTokenTests {
+
+        @BeforeAll
+        public static void beforeAll(final @NonNull TestLifecycle lifecycle) {
+            lifecycle.doAdhoc(
+                    fungibleToken.authorizeContracts(numericContractComplex),
+                    nft.authorizeContracts(numericContractComplex),
+                    alice.transferHBarsTo(numericContractComplex, ONE_HUNDRED_HBARS),
+                    numericContractComplex.getBalance().andAssert(balance -> balance.hasTinyBars(ONE_HUNDRED_HBARS)));
+        }
+
+        @HapiTest
+        @DisplayName("when using createFungibleTokenWithCustomFees with FixedFee")
```

Comment:
test "display names" throughout this PR are excellently descriptive, TY!

Reply:
Thanks! I think the desc text is usually easier to read than the method name, so it gives clearer context, IMO.